pyproject.toml

```
@@ -2,7 +2,7 @@
 name = "sphinx-needs"
 
 # !! Don't miss updates in sphinx_needs.__version__, changelog.rst, and .github/workflows/docker !!!
-version = "4.1.0"
+version = "4.2.0"
```

```suggestion
version = "4.1.0"
```

you don't need to change the version in a PR